### PR TITLE
Popover and context menu border unification & lighter popovers for dark variant

### PR DESCRIPTION
**Pull request by Feichtmeier**
Closes https://github.com/ubuntu/yaru/issues/783

Let's see if the testing snap works again ;) @madsrh 
Otherwise I post screenshots again

peek 2018-09-05 22-28

### CODE REVIEW

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr785`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr785` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

File: gtk/src/gtk-3.0/_colors.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@ $link_visited_color: if($variant == 'light', darken($link_color, 20%), lighten($
 $borders_edge: if($variant == 'light', rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.4));
 $top_hilight: $borders_edge;
 $dark_fill: if($variant == 'light',#CDCDCD, $ash);
-$menu_color: if($variant == 'light', $base_color, darken($base_color,5%));
+$menu_color: if($variant == 'light', $base_color, lighten($bg_color,6%));
```
Comment (clobrano) [inline gtk/src/gtk-3.0/_colors.scss]: space after comma ;)

**Comment by madsrh** [comment]
@Feichtmeier @clobrano are you happy with the current? 

I'm asking because I'm hoping you will say "_Perhaps it should have been a bit brighter_" 😉 

This PR is much better then what we had, but I think that if it's even brighter you get that illusion that it's on top / in front / popping out. We already have that effect with this PR, but just not as emphasized as I would like.

Too much ❔ 

[Current merged PR](https://user-images.githubusercontent.com/3986894/45166453-553e8800-b1f7-11e8-8788-65a1342b4568.png) vs. brighter:
image

**Comment by clobrano** [comment]
I fear that this bright ruin a bit the effect with check/radio buttons, but I should try on real system

**Pull request merged**